main.go

```diff
@@ -817,7 +817,7 @@ func main() {
 			rulesService:         rulesService,
 			jobsService:          jobsService,
 			versionServiceClient: versionService,
-			minio:                &minio.Service{},
+			minio:                minio.New(logrus.WithField("component", "minio-client")),
```

Comment:
Usually we don't pass logrus entry as parameter, we can create field in `New` method

Reply:
done